kafka/README.md

```diff
@@ -55,7 +76,10 @@ channels:
           clientId:
             type: string
             enum: ['myClientId']
-          bindingVersion: '0.1.0'
+          schemaIdLocation: 'payload'
+          schemaIdPayloadEncoding: 'apicurio-new'
+          schemaLookupStrategy: 'TopicIdStrategy'
```

Comment:
@lbroudoux thanks for this PR! I'm looking into some bindings for kafka topics too.

regarding schemaLookupStrategy: if its a class, then its not a freeform string, but an enum? is this [Redhat specific](https://access.redhat.com/documentation/en-us/red_hat_integration/2020-q4/html/getting_started_with_service_registry/using-kafka-client-serdes#registry-serdes-concepts-strategy-registry) (couldn't find TopicIdStrategy anywhere else)?

why not specifying the schema name and version for key and value?

Reply:
Yes, you're right : `TopicIdStrategy` is one of the strategies used by Apicurio Registry. While this information is useful, I thought it would be cumbersome to manage everything as enum value or FQDN classes...

About using schema name and version, I don't actually know if it's a current option among Ser/Deser libraries ... Maybe it could be another approach but it could be nice specifying a range or inclusion/exclusion pattern for versions in order to support bug fixes.